app/assets/javascripts/backbone/views/calendar_view.coffee

```diff
@@ -1,6 +1,28 @@
 # Common functions for calendars application-wide.
 class MS.Views.CalendarView extends Backbone.View
 
+  events:
+    'click .cal-step': 'stepClick'
+
+  prepareVariables: (params) ->
+    @$calendar = @$('#calendar')
+    @stepModal = params.stepModal
+
+  prepareDefaultSettings: (params, context) ->
```

Comment:
This is a neat refactor. Well done! Here are a few ways to make it simpler:

* Remove this method and just set `defaultSettings` inside the `initializeCalendar` method. Doesn't even have to be an instance variable.
* Pass the special settings as an argument to the `initializeCalendar` method instead of using an instance var.